demo/docker-compose.yml

```diff
@@ -80,38 +80,28 @@ services:
     networks:
       - tmail
 
-  krakend:
-    depends_on:
-      - sso.example.com
-      - tmail-backend
-    container_name: krakend
-    image: devopsfaith/krakend:latest
-    ports:
-      - "8090:8090"
-    volumes:
-      - ./krakend/krakend.json:/etc/krakend/krakend.json
-    networks:
-      - tmail
-
-  jwt-revoker:
-    build:
-      context: ./jwt-revoker
-    container_name: jwt-revoker
-    depends_on:
-      - krakend
-    networks:
-      - tmail
-
   apisix:
     container_name: apisix.example.com
-    image: apache/apisix:3.2.0-debian
+    image: linagora/apisix:3.2.0-debian-javaplugin
     volumes:
       - ./apisix/conf/apisix.yaml:/usr/local/apisix/conf/apisix.yaml
       - ./apisix/conf/config.yaml:/usr/local/apisix/conf/config.yaml
+      - ./apisix/tmail-apisix-plugin-runner/target/tmail-apisix-plugin-runner-0.0.1-SNAPSHOT.jar:/usr/local/apisix/token_revoker_plugin.jar
     networks:
       - tmail
     ports:
       - "9080:9080/tcp"
+      - "8081:8081"
+
+  redis:
+    container_name: redis.example.com
```

Comment:
Add this into /etc/hosts instruction?

Reply:
not need, backend uses it, end-user doesn't need to know it.